src/troute-network/troute/hyfeature_network_utilities.py

```diff
@@ -35,7 +38,27 @@ def build_forcing_sets(
         raise AssertionError("Aborting simulation because the nexus_input_folder:", qlat_input_folder,"does not exist. Please check the the nexus_input_folder variable is correctly entered in the .yaml control file") from None
 
     forcing_glob_filter = forcing_parameters.get("nexus_file_pattern_filter", "*.NEXOUT")
-  
+
+    if forcing_glob_filter=="nex-*":
+        print("Reformating qlat nexus files as hourly binary files...")
+        binary_folder = forcing_parameters.get('binary_nexus_file_folder', None)
+        nexus_files = nexus_input_folder.glob(forcing_glob_filter)
+
+        #Check that directory/files specified will work
+        if not binary_folder:
+            raise(RuntimeError("No output binary qlat folder supplied in config"))
+        elif not os.path.exists(binary_folder):
+            raise(RuntimeError("Output binary qlat folder supplied in config does not exist"))
+        elif len(os.listdir(binary_folder)) != 0:
+            raise(RuntimeError("Output binary qlat folder supplied in config is not empty"))
+
```

Comment:
All files in binary_folder should be cleanly deleted for a new simulation.  When Delete option is used, one hidden file, i.e., .ipynb_checkpoints,  still remains to give the error of line 53.  It is recommended to use "rm -r * .*" to cleanly delete all files.  Other than this, this PR successfully ran with nextgen-generated csv files.

Reply:
This new change at line 52 solved my initial issue. Well done!!